.github/workflows/manual-decommission-testnet-l1.yml

```diff
@@ -42,18 +42,37 @@ jobs:
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
 
+      - name: 'Print out resource list'
+        uses: azure/CLI@v1
+        with:
+          inlineScript: |
+            if [ -n "${{ vars.AZURE_DEPLOY_GROUP_L1 }}" ]; then
+              az resource list --tag ${{ vars.AZURE_DEPLOY_GROUP_L1 }}=true --query '[]."id"' -o tsv 
+            else
+              echo "Warning filter condition is blank or empty"
+              exit 1  
+            fi
+
+      - name: 'Add a cool off sleep period if cancel needed'
+        run: |
+          sleep 20
+
       # This will fail some deletions due to resource dependencies (i.e. you must first delete the vm before deleting the disk)
       - name: 'Delete deployed VMs'
         uses: azure/CLI@v1
         continue-on-error: true
         with:
           inlineScript: |
-            $(az resource list --tag ${{ vars.AZURE_DEPLOY_GROUP_L1 }}=true --query '[]."id"' -o tsv | xargs -n1 az resource delete --verbose -g Testnet --ids) || true
+            if [ -n "${{ vars.AZURE_DEPLOY_GROUP_L1 }}" ]; then
```

Comment:
this is just being double protective, as the workflow should have exited if the filter criteria is blank

Reply:
nice